### DISCUSSION TITLE
Move search box to the side alignment

### DISCUSSION THREAD

**Discussion by BonesJD** [valinet/ExplorerPatcher#2564] [Q&A]
I use an ultrawide monitor for streaming where I use the left 2560x1440 for the stream and the right 880x1440 for stuff that I don't want on stream, including the task bar, start menu and I would wish to have the search box there too.

As you can see in the screen shots, the start menu successfully aligns to the right with the task bar but the search box still open at the bottom middle. Is there a way to fix this already? if not, is there a chance of fixing this in an update if I put in a feature request?

![Screenshot (272)](https://github.com/valinet/ExplorerPatcher/assets/62482002/ef11eb10-5743-425e-b0a5-26fe197986d4)
![Screenshot (273)](https://github.com/valinet/ExplorerPatcher/assets/62482002/36e0d67d-f705-4802-b5d1-cadb6b0af7ec)

**Comment by pyrates999** [comment]
duplicate of: https://github.com/valinet/ExplorerPatcher/issues/1803

Please close this.
